cluster/etcd_service_discovery.go

```diff
@@ -108,6 +110,16 @@ func (sd *etcdServiceDiscovery) configure() {
 	sd.grantLeaseMaxRetries = sd.config.GetInt("pitaya.cluster.sd.etcd.grantlease.maxretries")
 	sd.grantLeaseInterval = sd.config.GetDuration("pitaya.cluster.sd.etcd.grantlease.retryinterval")
 	sd.shutdownDelay = sd.config.GetDuration("pitaya.cluster.sd.etcd.shutdown.delay")
+	sd.serverTypesBlacklist = sd.config.GetStringSlice("pitaya.cluster.sd.etcd.servertypeblacklist")
+	sd.syncServersParallelism = sd.config.GetInt("pitaya.cluster.sd.etcd.syncserversparallelism")
+
+	if sd.syncServersParallelism == 0 {
```

Comment:
You should add all the configs here https://github.com/topfreegames/pitaya/blob/master/config/config.go
Also, by doing that you will not need this if, because you'll already specify the default value there.